### DISCUSSION TITLE
✨ Guide - How to migrate Airbyte connector to Low-code CDK

### DISCUSSION THREAD

**Discussion by shivam096** [airbytehq/airbyte#29997] [Show and tell]
Hello, everyone!

I wrote a blog post that takes an deep dive into the world of Custom Connectors within Airbyte. 

Here's the link to the blog:  https://shivam096.hashnode.dev/guide-written-by-a-novice-creator-on-how-to-migrate-airbyte-connector-to-low-code

But hey, this isn't a one-way conversation. I'm genuinely eager to hear what's in your minds too. Let's make this a two-way street. Share your:

* Thoughts on the blog. Any suggestions or methods to approach the process differently
*  Burning questions that might have popped up as you read through my blog 

I am thrilled to anticipate all the insightful conversations that will unfold. 

Lets continue the chain in the discussion

**Also if you liked the blog please give this GitHub discussion an upvote as it would help me with the contest**  :)

**Comment by archith23** [comment]
This blog gets straight to the point with minimal fluff and written in a way that's easy to understand even for a newbie like me 👍🏽

**Reply by shivam096** [reply]
Thanks Archit and I’m glad that this helped.
